source/blog/2016-02-24-devconf-2016.html.md

```diff
@@ -0,0 +1,48 @@
+---
+title: My Devconf.CZ 2016 experience
+author: ykaul
+tags: community, news, blog
+date: 2016-03-02 15:00:00 CET
+---
+
+On the first weekend of February I've had the pleasure of attending DevConf.CZ 2016, which took place in the wonderful city of Brno, CZ.
```

Comment:
* I've had the pleasure > I had the pleasure
* Brno, CZ. > Brno, Czech Republic.